bagel/tests/test_cli_pheno.py

```diff
@@ -81,6 +81,48 @@ def test_invalid_inputs_are_handled_gracefully(
         assert substring in str(e.value)
 
 
+@pytest.mark.parametrize(
+    "portal",
+    [
+        "openneuro.org/datasets/ds002080",
+        "https://openneuro",
+        "not a url",
+        "www.github.com/mycoolrepo/mycooldataset",
+    ],
+)
+def test_invalid_portal_uris_produces_stderr(
```

Comment:
For clarity, rename test: 
```suggestion
def test_invalid_portal_uris_produces_error(
```